# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/PCookieStore.ipdl
# Commit: 0d57b435d872
# Full Hash: 0d57b435d872751c2a26879fb04bf8745314e37a
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-12 09:23:07
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 8 - CookieStore API - e10s support, r=smaug,cookie-reviewers,timhuang
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D217574
# ==============================================================================

diff -r a63a5bcb0524 -r 0d57b435d872 dom/cookiestore/PCookieStore.ipdl
--- a/dom/cookiestore/PCookieStore.ipdl	Wed Sep 11 16:26:57 2024 +0000
+++ b/dom/cookiestore/PCookieStore.ipdl	Wed Sep 11 16:26:57 2024 +0000
@@ -6,6 +6,7 @@
 
 using mozilla::OriginAttributes from "mozilla/ipc/BackgroundUtils.h";
 using struct mozilla::void_t from "ipc/IPCMessageUtils.h";
+using struct nsID from "nsID.h";
 
 namespace mozilla {
 namespace dom {
@@ -44,13 +45,15 @@
                    int64_t expires,
                    nsString path,
                    int32_t sameSite,
-                   bool partitioned) returns (bool v);
+                   bool partitioned,
+                   nsID operationId) returns (bool waitForNotification);
 
   async DeleteRequest(nsString domain,
                       OriginAttributes attrs,
                       nsString name,
                       nsString path,
-                      bool partitioned) returns (bool v);
+                      bool partitioned,
+                      nsID operationID) returns (bool waitForNotification);
 
   async Close();
 